server/liveblog/blogs/blogs.py

```diff
@@ -133,6 +136,13 @@ def on_create(self, docs):
             prefs = global_prefs.copy()
             prefs.update(doc.get('blog_preferences', {}))
             doc['blog_preferences'] = prefs
+            doc['theme_settings'] = {}
```

Comment:
It initializes the empty dict where the themes settings will be saved on blog level. Yeah, I think we could live without it though :)